src/server/api/routes/cvs.router.js

```diff
@@ -23,7 +23,7 @@ const cvsController = require('../controllers/cvs.controller');
  */
 router.get('/', (req, res, next) => {
   cvsController
-    .getCvs()
+    .getCvs(req.query.title, req.query.limit)
```

Comment:
What if i have neither title or limit in the query parameter. Does this still work?

Reply:
yes if there is no any parameter it returns the all cvs